File: app/src/androidTest/java/com/example/rafaelanastacioalves/moby/util/RestServiceTestHelper.java
package com.example.rafaelanastacioalves.moby.util;

import android.content.Context;

import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStream;
import java.io.InputStreamReader;

/**
 * Created by rafaelanastacioalves on 28/03/18.
 */

public class RestServiceTestHelper {
    public static String getStringFromFile(Context context, String filpePath) throws IOException {
        final InputStream stream = context.getResources().getAssets().open(filpePath);
        String ret = convertStreamToString(stream);
        stream.close();
        return ret;
    }

    private static String convertStreamToString(InputStream is) throws IOException {
        BufferedReader reader = new BufferedReader(new InputStreamReader(is));
        StringBuilder sb = new StringBuilder();
        String line;
        while ((line = reader.readLine()) != null) {
            sb.append(line).append("\n");
        }
        reader.close();
        return sb.toString();    }
}

File: app/src/androidTest/java/com/example/rafaelanastacioalves/moby/util/RestServiceTestHelper.kt
package com.example.rafaelanastacioalves.moby.util

import android.content.Context

import java.io.BufferedReader
import java.io.IOException
import java.io.InputStream
import java.io.InputStreamReader


object RestServiceTestHelper {
    @Throws(IOException::class)
    fun getStringFromFile(context: Context, filpePath: String): String {
        val stream = context.resources.assets.open(filpePath)
        val ret = convertStreamToString(stream)
        stream.close()
        return ret
    }

    @Throws(IOException::class)
    private fun convertStreamToString(`is`: InputStream): String {
        val reader = BufferedReader(InputStreamReader(`is`))
        val sb = StringBuilder()
        var line: String?
        line = reader.readLine()
        while (line != null) {
            sb.append(line).append("\n")
            line = reader.readLine()
        }
        reader.close()
        return sb.toString()
    }
}
